OneSignalSDK/onesignal/build.gradle

```diff
@@ -56,6 +57,10 @@ dependencies {
     // firebase-messaging:17.6.0 is the max version since we still have code looking for FirebaseInstanceIdService
     api 'com.google.firebase:firebase-messaging:[10.2.1, 17.3.99]'
 
+    implementation 'com.huawei.agconnect:agconnect-core:1.3.1.300'
```

Comment:
These should be `compileOnly` so they are not included in our POM file as required dependencies.

Reply:
Fix in PR #1043 and has been merged into this PR already.